go.mod

```diff
@@ -5,11 +5,11 @@ go 1.15
 require (
 	cloud.google.com/go v0.56.0 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/golang/protobuf v1.4.2
+	github.com/golang/protobuf v1.5.2
 	github.com/jhump/protoreflect v1.10.2
 	golang.org/x/text v0.3.7 // indirect
 	google.golang.org/grpc v1.37.0
-	google.golang.org/protobuf v1.25.1-0.20200805231151-a709e31e5d12
+	google.golang.org/protobuf v1.26.0
```

Comment:
This gets pulled along for the ride because required by the newer version of `github.com/golang/protobuf`... relevant changelog here: https://github.com/protocolbuffers/protobuf-go/releases/tag/v1.26.0